frontend/src/shared/helpers.tsx

```diff
@@ -535,36 +475,7 @@ export function prepareCohortDataForAgGrid(
   cohortsListQueryResult: CohortsListQuery,
   filterModel: IServerSideGetRowsRequest["filterModel"]
 ) {
-  let newCohorts = cohortsListQueryResult.cohorts.map((cohort) => {
-    const samples = cohort.hasCohortSampleSamples;
-    const cohortCompletes = cohort.hasCohortCompleteCohortCompletes;
-
-    const allSamplesBilled =
-      samples.length > 0 &&
-      samples?.every((sample) => {
-        return sample.hasTempoTempos?.[0].billed === true;
-      });
-
-    const latestCohortDeliveryDate = cohortCompletes?.[0];
-
-    return {
-      cohortId: cohort.cohortId,
-      totalSamples: cohort.hasCohortSampleSamplesConnection.totalCount,
-      smileSampleIds: samples.map((s) => s.smileSampleId),
-      billed: allSamplesBilled === true ? "Yes" : "No",
-      initialCohortDeliveryDate: formatDate(
-        cohortCompletes?.slice(-1)[0]?.date
-      ),
-      completeDate: formatDate(latestCohortDeliveryDate?.date),
-      endUsers: latestCohortDeliveryDate?.endUsers,
-      pmUsers: latestCohortDeliveryDate?.pmUsers,
-      projectTitle: latestCohortDeliveryDate?.projectTitle,
-      projectSubtitle: latestCohortDeliveryDate?.projectSubtitle,
-      status: latestCohortDeliveryDate?.status,
-      type: latestCohortDeliveryDate?.type,
-    };
-  });
-
```

Comment:
Removed code that flatten data on the frontend. This step is now done at the GraphQL server level.